cli/dts/lib.deno.ns.d.ts

```diff
@@ -2298,6 +2300,36 @@ declare namespace Deno {
     | "SIGXCPU"
     | "SIGXFSZ";
 
+  /** Registers the given function as a listener of the given signal event.
+   *
+   * ```ts
+   * Deno.addSignalListener("SIGTERM", () => {
+   *   console.log("SIGTERM!")
+   * });
+   * ```
+   *
+   * NOTE: This functionality is not yet implemented on Windows.
+   */
+  export function addSignalListener(signal: string, handler: () => void): void;
```

Comment:
```suggestion
  export function addSignalListener(signal: Signal, handler: () => void): void;
```

I think this should use the `Signal` type definition. It makes for much better editor completions.